File: kids-with-the-greatest-number-of-candies.py
class Solution:
  def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
    result = []
    maximum = max(candies)
    for i in range (0, len(candies)):
      if (candies[i] + extraCandies >= maximum): result.append(True)
      else: result.append(False)
    return result

File: kids-with-the-greatest-number-of-candies.rs
impl Solution {
    pub fn kids_with_candies(candies: Vec<i32>, extra_candies: i32) -> Vec<bool> {
        let mut v: Vec<bool> = vec![];

        let max = candies.iter().max().unwrap();

        for n in 0..candies.len() {
            v.push(candies[n] + extra_candies >= *max);
        }

        return v;
    }
}
